ui/app/AppLayouts/Wallet/popups/filterSubMenus/ActivityTypeFilterSubMenu.qml

```diff
@@ -0,0 +1,86 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import StatusQ.Popups 0.1
+
+import "../../controls"
+
+StatusMenu {
```

Comment:
I found few little issues here:
* Clicking the delegate (outside the checkbox) doesn't pass click to checkbox.
* Height is wrong (https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=12304-251066&t=1IKHvElko6uA7Yoq-4) Points it should be 44. When I did run it, it was 64. The difference is too big.
![image](https://github.com/status-im/status-desktop/assets/11396062/baee668d-d60a-40f5-99ef-27cb2b85bfbc)